### PR TITLE
SONAR-9763 performance hotspots in LTS upgrade

### PR DESCRIPTION
**Pull request by simonbrandhof**

### CODE REVIEW

File: server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v60/PopulateAnalysisUuidOnMeasures.java [hunk 1]
```diff
@@ -34,32 +36,43 @@ public PopulateAnalysisUuidOnMeasures(Database db) {
...
       "where m.analysis_uuid is null"
     );
-    massUpdate.update("update project_measures set analysis_uuid=? where snapshot_id=? and analysis_uuid is null");
+    massUpdate.update("update project_measures set analysis_uuid=? where snapshot_id = ? and analysis_uuid is null");
     massUpdate.rowPluralName("measures");
-    massUpdate.execute(PopulateAnalysisUuidOnMeasures::handle);
+    massUpdate.execute((row, update) -> handleRow(row, update, rootSnapshotUuids));
+  }
+
+  private Map<Long, String> loadRootSnapshotUuids(Context context) throws SQLException {
```
Comment (SonarTech) [inline server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v60/PopulateAnalysisUuidOnMeasures.java]: MAJOR Make "loadRootSnapshotUuids" a "static" method. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by sns-seb** [comment]
@simonbrandhof has QA on Oracle run on this PR?
If so, LGTM after fixing @SonarTech 's comment
otherwise, I think it should run

**Comment by simonbrandhof** [comment]
No need to run QA. This version is heavily tested on all databases for two weeks !

**Pull request closed without merging**